### ISSUE TITLE
 TypeError: Cannot read property 'close' of null

### ISSUE THREAD

**Issue by jochenonline** [sockjs/sockjs-node#201]
Ten seconds after a socket connection has closed, we get the following error occurs:

```
TypeError: Cannot read property 'close' of null
at WebSocketReceiver.heartbeat_timeout (/home/red/testclient/0000/server/node_modules/sockjs/lib/trans-websocket.js:157:26)
at null._onTimeout (/home/red/testclient/0000/server/node_modules/sockjs/lib/trans-websocket.js:75:24)
at Timer.listOnTimeout (timers.js:92:15)
```
Do you have any hint what could be the reason for this?

**Comment by dagingaa** [comment]
Seems to be the same issue as reported in #185. Reverting to 0.3.15 should work.

**Comment by brycekahle** [comment]
Fixed by #179

**Comment by brycekahle** [comment]
Released in 0.3.17
